append_lifecycle.go

```diff
@@ -108,6 +108,7 @@ func NewAppendOptions() *AppendOptions {
 		batchMaxSize:           DefaultBatchMaxSize,
 		batchMaxAge:            DefaultBatchMaxAge,
 		entriesPath:            layout.EntriesPath,
+		bundleIDHasher:         defaultIDHasher,
```

Comment:
I couldn't leave a comment on L100. Should the `defaultIDHasher` in `WithAntispam()` be updated to `o. bundleIDHasher`?

Reply:
Yes - great spot, thanks!